src/H5Fint.c

```diff
@@ -2210,6 +2227,9 @@ H5F_open(const char *name, unsigned flags, hid_t fcpl_id, hid_t fapl_id)
     /* Success */
     ret_value = file;
 
+#if 1 /*Kent: write to the log file when H5F_open ends. Tested, can be commented out if necessary.*/
+    H5F_POST_VFD_SWMR_LOG_ENTRY(file, 1, "File open ends");
```

Comment:
We should have #defines for the entry_type_code parameters, not just integer constants.

Suggest you put the definitions of these #defines in the same file as your macro wrappers for the logging function proper.

Reply:
On further review, probably best to put them adjacent to your declaration of H5Fvfd_swmr_log_tags.

Reply:
Done.